src/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -56,13 +56,9 @@ const isValidAddress = (protocolVersion: ProtocolVersion, addressStr: string) =>
 const isReservedAddress = (ip: string, protocolVersion: ProtocolVersion) => {
   try {
     if (protocolVersion === ProtocolVersion.ipv4) {
-      return ip === RESERVED_LOCAL_HOST_IPS.ipv4 || ip === RESERVED_CATCH_ALL_IPS.ipv4;
+      return RESERVED_LOCAL_HOST_IPS.ipv4.includes(ip) || RESERVED_CATCH_ALL_IPS.ipv4.includes(ip);
```

Comment:
Based on the previous comment, create another function `isReservedIpv6Address` and check all possibilities there.

Reply:
Let's talk about this on Monday

Reply:
Ok, new function created